docs/guides/telemetryservice/pushtoTDEngine.md

```diff
@@ -28,10 +28,21 @@ spec:
 - `dbTable` is the table name of the database
 - `dbType` is the type of the database
   
+
 :::note
 If you have multiple telemetry services, you can write them in one file and split them in one file using `---`.
 :::
 
+:::note use secret to store password
+
+Create a  `Secret` with `telemetry_service_sql_pwd` field filled by your password, and it will overwrite the `SQLSetting.secret` in `TelemetryService`.
```

Comment:
can we also add the process(from UI to backend) of how to store the secret and how to get that secret in this k8s Secret service?

Reply:
ok, now have added the process of how to get and use `Secret`.